stories/Loaders/Examples/ProgressBar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ProgressBar from '../../../components/ProgressBar';
+
+export default [
+  {
+    code: '<ProgressBar />',
+    component: <ProgressBar />,
+  },
+  {
+    code: '<ProgressBar progressPercent={30} />',
+    component: <ProgressBar progressPercent={30} />,
```

Comment:
On the unit test you should add a test passing props like that.

Reply:
Probably because the tests are outdated with the master branch.